Tools/Scripts/webkitpy/w3c/test_exporter.py

```diff
@@ -316,7 +331,7 @@ def create_wpt_pull_request(self, remote_branch_name, title, body):
     def delete_local_branch(self, *, is_success=True):
         if self._options.clean and (is_success or self._options.clean_on_failure):
             _log.info('Removing local branch ' + self._branch_name)
-            self._run_wpt_git(['checkout', self._wpt_repo.default_branch])
+            self._run_wpt_git(['checkout', self._wpt_repo.default_branch], capture_output=True)
```

Comment:
Why are we capturing output here? If we just want it to not appear, we should just use `--quiet`.